wcfsetup/install/files/lib/system/comment/manager/ICommentManager.class.php

```diff
@@ -132,6 +131,11 @@ public function getTitle($objectTypeID, $objectID, $isResponse = false);
      */
     public function isAccessible($objectID, $validateWritePermission = false);
 
+    /**
+     * Returns true if the user may read content identified by object type id and object id.
+     */
+    public function canViewObject(int $objectID, UserProfile $user): bool;
```

Comment:
There is no default implementation for this method, because it cannot be provided meaningfully, thus is break of the backwards compatibility. You MUST use a new interface for this!